media/css/foundation/foundation.less

```diff
@@ -2,258 +2,426 @@
+.billboard,
+.content {
+    width: @widthDesktop - (@gridGutterWidth * 2);
+    margin-left: auto;
+    margin-right: auto;
 }
 
-a.go {
-  .trailing-arrow;
+.mobile-list-table {
+    margin-bottom: 1.5em;
 }
-
-#video-description,
-.video-description {
-  margin: 1em @gridGutterWidth 1.5em;
-  h3 {
-    .font-size(20px);
-    letter-spacing: normal;
-  }
-  h4 {
-    .font-size(@largeFontSize);
-    letter-spacing: normal;
-  }
-  .download {
-    margin: 0 0 .5em;
-    .font-size(@smallFontSize);
-  }
+.mobile-list-table thead td {
+    background: #f5f5f5;
+    padding: 0.25em;
+    font-weight: bold;
 }
-
-.timestamp {
-  .font-size(20px);
-  letter-spacing: normal;
+.mobile-list-table tbody td {
+    border-bottom: 1px dotted #ccc;
+    padding: 0.5em 0.25em 2em;
+    vertical-align: top;
 }
 
-.nav-paging {
-  clear: both;
-  list-style: none;
-  .font-size(@largeFontSize);
-  font-style: italic;
-  margin: 15px 0;
-  padding: 0;
-  overflow: hidden;
-  li {
-    margin: 0;
-  }
-}
+.page-header {
+    background: url('/media/img/foundation/banner/foundation_hero-image-small.jpg') center bottom no-repeat;
+    .background-size(cover);
+    padding-top: 300px;
 
-.nav-paging .prev {
-  float: left;
-  width: 45%;
-  text-align: left;
-}
+    @media only screen and (min-width: @breakDesktop) {
+        background-image: url('/media/img/foundation/banner/foundation_hero-image-large.jpg');
+    }
 
-.nav-paging .prev a:before {
-  .font-size(24px);
-  content: "\00AB\00A0";
-}
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        background-image: url('/media/img/foundation/banner/foundation_hero-image-medium.jpg');
+    }
 
-.nav-paging .next {
-  float: right;
-  width: 45%;
-  text-align: right;
-}
+    @media only screen and (max-width: @breakTablet) {
+        padding-top: 150px;
+        background-image: url('/media/img/foundation/banner/foundation_hero-image-small.jpg');
+    }
+
+    @media only screen and (min-width: @breakMobileLandscape) and (max-width: @breakTablet) {
+        background-image: url('/media/img/foundation/banner/foundation_hero-image-medium.jpg');
+    }
 
-.nav-paging .next a:after {
-  .font-size(24px);
-  content: "\00A0\00BB";
+    h1 {
+        background: @mozillaRed;
+        color: #fff;
+        margin-bottom: 0;
+        padding: .35em 0;
+        text-shadow: none;
+    }
+
+    .content {
+        .border-box;
+        display: block;
+        padding: 0 10px;
+    }
 }
 
-.return-to-top {
-  .font-size(@smallFontSize);
+.content-main {
+    .box-shadow(0 0 0 1px #fff inset);
+    .clearfix;
+    .open-sans;
+    background: #fff;
+    border-bottom: 1px solid #ddd;
+    display: block;
+    padding: 40px @gridGutterWidth;
+    position: relative;
+
+    .main-column {
+        .span(8);
+    }
+
+    .sidebar {
+        .offset(1);
+    }
+
+    li + li {
+        margin-top: 1em;
+    }
+
+    .sidebar li {
+        padding: 0.5em 0;
+    }
+
+    ul.links {
```

Comment:
Ok, I was hesitant to touch things here because removing CSS can be such a house of cards